src/main/java/io/micrometer/newrelic/NewRelicRegistry.java

```diff
@@ -260,7 +260,8 @@ private MetricBatchSender createMetricBatchSender() {
       MetricBatchSenderBuilder metricBatchSenderBuilder =
           MetricBatchSender.builder()
               .apiKey(config.apiKey())
-              .httpPoster(new MicrometerHttpPoster(httpSender));
+              .httpPoster(new MicrometerHttpPoster(httpSender))
+              .secondaryUserAgent("NewRelic-Micrometer-Exporter", implementationVersion);
```

Comment:
Shouldn't the user agent be `NewRelic-Java-Micrometer` according to the spec format  guidelines of `<author>-<language>-<exporter_name>`? 

https://github.com/newrelic/newrelic-exporter-specs/blob/master/Common.md

Reply:
Is there a non-java micrometer?  ;)

Reply:
No there's not, but I'd tend to favor the consistency of adhering to a spec rather than doing something different just for the sake of it. I think it makes for a more consistent experience for both customers and our support teams. I like patterns, patterns are good. ;)

Reply:
also, java is totally redundant because it's in the first part of the user-agent from the sdk itself. I think we should change that spec.

Reply:
I put in a PR to change the spec. We'll see what happens. :)

Reply:
![excellent](https://media.giphy.com/media/8fen5LSZcHQ5O/giphy.gif)

Reply:
ok, spec PR has merged!!!

Reply:
Excellent, looks good! I'll go ahead and resolve this conversation and approve the PR.